integration-tests/__tests__/__snapshots__/empty-describe-with-hooks.test.js.snap

```diff
@@ -0,0 +1,142 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`hook in empty describe 1`] = `
+Object {
+  "rest": "FAIL __tests__/hook-in-empty-describe.test.js
+
+
+  ● Test suite failed to run
+
+    thrown: \\"Invalid: beforeEach() in a describe block with no tests.\\"
+
+       7 | 
+       8 | describe('a block', () => {
+    >  9 |   beforeEach(() => {});
+         |   ^
+      10 | });
+      11 | 
+      12 | describe('another block with tests', () => {
+
+      at packages/jest-circus/build/index.js:33:17
```

Comment:
@SimenB Is there anything special that I need to do to filter out this line of the stack trace?

Reply:
I was a bit lazy in #6281 and didn't add all the `Error.captureStackTrace`s I should. I just added it to test, should be added to all hooks as well